### DISCUSSION TITLE
How to directly use the client library to run the kubectl get --raw='/readyz?verbose' command.

### DISCUSSION THREAD

**Discussion by lirany1** [fabric8io/kubernetes-client#4768] [Q&A]
Tried both:
```
val client = DefaultKubernetesClient()
val response = client.nodes().handleRaw("/readyz?verbose")
println(response)
```

```
val client = DefaultKubernetesClient()
val response = client.nodes().get("--raw", "/readyz?verbose")
println(response)
```

not working for 6.3.1 version any suggestion?

**Comment by shawkins** [comment]
On this post and the other some of the things you are trying are not part of the public api.  You should only use the public methods.  If something is not exposed, but you'd like it to be, that will take an enhancement.  

There is no high level way yet that is publicly available to issue a raw request.
